sklearn/covariance/_shrunk_covariance.py

```diff
@@ -445,6 +447,11 @@ class LedoitWolf(EmpiricalCovariance):
     array([ 0.0595... , -0.0075...])
     """
 
+    _parameter_constraints = {
+        **EmpiricalCovariance._parameter_constraints,
+        "block_size": [Interval(Integral, 0, None, closed="left")],
```

Comment:
`block_size=0` will lead to a division by 0.

```suggestion
        "block_size": [Interval(Integral, 1, None, closed="left")],
```

Reply:
thanks a lot, I updated it